### ISSUE TITLE
Guest Agent - This program doesn't seem to be run under Pyrebox.

### ISSUE THREAD

**Issue by xorhex** [Cisco-Talos/pyrebox#56]
The agent can't detect that it's running in a pyrebox VM.  Am I missing something?

![screenshot from 2018-07-01 11-58-30](https://user-images.githubusercontent.com/40742023/42137696-269c221a-7d26-11e8-8056-27a3d252f493.png)

Host OS: Ubuntu 18.04 LTS
Guest OS: Windows 7 Ultimate SP1 x86
Agent Name: win_agent_32.exe

#### pyrebox.conf
[Modues]
plugins.guest_agent: True
.
.
[AGENT]
name: win_agent_32.exe
conf: win_agent_32.exe.conf

#### win_agent_32.exe.conf
[BUFFER]
BufferOffset:  25568
BufferSize:  4096

Start machine using:`
./pyrebox-i386 -monitor stdio -m 4096 -usb -device usb-tablet -drive file=images/win7sp1.qcow2,index=0,media=disk,format=qcow2,cache=unsafe -vnc 127.0.0.1:0`

```
[*] Loading python component initialization script
[*] Platform: i386-softmmu
[*] Starting python module initialization
[*] Reading configuration from 'pyrebox.conf'
[*] Searching for KDBG...
[*] Initializing scripts...
[*]  Loading python module plugins.guest_agent
[plugins.guest_agent] [*]    Initializing guest_agent plugin
[*] Finished python module initialization
QEMU 2.12.0 monitor - type 'help' for more information
```

**Comment by xorhex** [comment]
Hello xabiugarte, 

Yeah, my bad, that was a typo.

Copied the guest_agent.py over and re-ran.  

![screenshot from 2018-07-02 07-33-20-edited](https://user-images.githubusercontent.com/40742023/42173319-7e2ae8a6-7dd3-11e8-87e2-6d0c28524583.png)

**Comment by xabiugarte** [comment]
Ok, I think I just realized about what's happening.

Can you try using just Win7SP1x86 as profile, instead of Win7SP1x86_23418?

If you can confirm me that it works with that configuration change, I will submit a patch to fix that.

**Comment by xorhex** [comment]
That did the trick!  I probably introduced it when coping the profile over from https://github.com/volatilityfoundation/volatility/wiki/2.6-Win-Profiles#profile-lists to create that config.

Thanks!!

**Comment by xabiugarte** [comment]
Perfect! Since this is a bug, I'll leave the issue open and fix it shortly.

**Comment by xabiugarte** [comment]
Fixed in commit 273406f38322cc425526cce141105ce70dc162dd. Closing issue.
